### PR TITLE
Anti spam bot shield

### PR DESCRIPTION
**Pull request by froger**

### CODE REVIEW

File: spec/decidim/services/decidim/spam_detection_service_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+require "spec_helper"
+
+describe Decidim::SpamSignal::SpamDetectionService::class do
+
+    let(:spam_detection_service) { Decidim::SpamSignal::SpamDetectionService }
+
+    it "valid?(content)" do
+      expect(spam_detection_service.valid?('https://Sexy')).to eq(false)
+      expect(spam_detection_service.valid?('blackdomain.gg')).to eq(false)
+      expect(spam_detection_service.valid?('lausanne.ch')).to eq(true)
```
Comment (froger) [inline spec/decidim/services/decidim/spam_detection_service_spec.rb]: hehe, sorry! But it makes no sense to check a whitelist domain on content validation. 

```
"blackdomain.gg/lausanne.ch" will be valid
```

I would suggest to remove completely logics of whitelist domain

**Review by froger — COMMENTED** [review]
Hi @rsdeus to me it's almost ready to merge. We should consider removing the logic of whitelist domains for now, and make a future optimisation with this. There are some use cases we need to consider.

File: app/jobs/decidim/spam_signal/scan_for_spam_job.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module Decidim
```
Comment (froger) [inline app/jobs/decidim/spam_signal/scan_for_spam_job.rb:1]: @rsdeus I think you can remove this file ;)

Comment (rsdeus) [reply app/jobs/decidim/spam_signal/scan_for_spam_job.rb:1]: I've removed `app/jobs/decidim/spam_signal/spam_signal_job.rb` instead

**Pull request closed without merging**